### ISSUE TITLE
Standalone desktop application?

### ISSUE THREAD

**Issue by baimafeima** [ncase/loopy#13]
Is there also a standalone desktop application of loopy capable to run on a Linux-based operating system?

**Comment by ncase** [comment]
If you download the repo and open up `index.html` in Firefox, it should work just fine locally!

(Chrome has a "problem" where it won't do AJAX calls on file://, because of cross-origin security whatever. If you want to use Chrome, you'll need to run a local server using [http-server](https://www.npmjs.com/package/http-server) or something)

**Comment by baimafeima** [comment]
Thanks, but I actually meant whether you have any plans to develop a standalone desktop version of loopy. The look and feel would be different from opening it in a browser.

**Comment by Ryuno-Ki** [comment]
What about putting it in Electron?

**Comment by baimafeima** [comment]
@Ryuno-Ki Yeah, why not. Probably the easiest way given it's browser-based?

**Comment by marktnoonan** [comment]
If users are confidant doing it, they can use Nativefier to create a standalone app from this site (or any other) that will run on their system.

**Comment by jimtobias** [comment]
I'm having trouble thinking of compelling use cases. I see you're interested in privacy -- is that it?
